scrapers/RealityLovers.py

```diff
@@ -0,0 +1,197 @@
+import json
+import sys
+import re
+import requests
+import time
+import base64
+
+# Set these cookies to the values from your RealityLovers browser session
```

Comment:
Is setting the cookies from browser actually needed ? I left it as it is and it seems to work ok?